src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -170,6 +178,16 @@ public function editAction($featureId, Request $request): Response
         ]);
     }
 
+    /**
+     * @param Request $request
```

Comment:
Missing security annotation here:
```suggestion
     * @AdminSecurity("is_granted('update', request.get('_legacy_controller'))")
     * @param Request $request
```